core/src/components/breadcrumb/breadcrumb.ios.scss

```diff
@@ -10,6 +10,12 @@
   --color-hover: #{$breadcrumb-ios-color-active};
   --color-focused: var(--color-active);
   --background-focused: #{$breadcrumb-ios-background-focused};
+
+  /**
+   * Main content should be prioritized
+   * on iOS, so we set max font size for breadcrumbs.
```

Comment:
Can we update this comment too to mention the min font size now?

Reply:
Addressed in https://github.com/ionic-team/ionic-framework/pull/27759/commits/11c4ff9ca9d364d6f04c38bc8196b3e468a003a4